packages/evm/src/verkleAccessWitness.ts

```diff
@@ -377,3 +389,57 @@ export class VerkleAccessWitness implements VerkleAccessWitnessInterface {
     }
   }
 }
+
+export const generateExecutionWitness = async (
+  stateManager: StatefulVerkleStateManager | StatelessVerkleStateManager,
+  accessWitness: VerkleAccessWitness,
+  parentStateRoot: Uint8Array,
+) => {
+  const trie = (stateManager as StatefulVerkleStateManager)['_trie'] as VerkleTree
+  const postStateRoot = await stateManager.getStateRoot()
```

Comment:
The helper accept `  stateManager: StatefulVerkleStateManager | StatelessVerkleStateManager` but it seems like only the StatefulVerkleStateManager would be valid input? Otherwise wouldn't `trie` and `getStateRoot` fail?